docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<p> </p><blockquote>Many problems in data science and engineering are well studied and there exist known parallel algorithms for making some desired transformation to some data. Many have corresponding CUDA solutions that may not exist as column level API in cuDF. To expose the ability to use these custom kernels, cuDF supports directly using custom cuda kernels written using <code>numba</code> on cuDF <code>Series</code> objects. In short, this means that if a user has knowledge of how to write a CUDA kernel in numba, they may simply pass cuDF <code>Series</code> objects to that kernel as if they were numba device arrays. Let's look at a basic example of how to do this.</blockquote><p> </p><p>Two observations here:</p><p> </p><ul><li>We should avoid categorizing users as coming from a data science or engineering background</li><li>It takes a bit of reading before I understand what this section is about.</li></ul><p> </p><p>I think we should be more direct here; something along the lines of:</p><p> </p><blockquote>In addition to the <code>Series.apply()</code> method for performing custom operations, you can also pass <code>Series</code> objects directly into [CUDA kernels written with Numba](https://numba.pydata.org/numba-doc/latest/cuda/kernels.html).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>